### PR TITLE
qmake try silent build

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: qgroundcontrol.pro [hunk 1]
```diff
@@ -92,6 +92,8 @@ QT += \
     bluetooth \
 }
 
+QT += silent
```
Comment (NaterGator) [inline qgroundcontrol.pro]: Should be CONFIG += silent

Comment (dagar) [inline qgroundcontrol.pro]: thanks

**Comment by dagar** [comment]
...and it breaks the windows build. Sounds like a job for tomorrow.

**Comment by NaterGator** [comment]
Only thing I can think of is https://ci.appveyor.com/project/mavlink/qgroundcontrol/build/master-2175#L231 is triggered by https://ci.appveyor.com/project/mavlink/qgroundcontrol/build/master-2175#L204

**Comment by DonLakeFlyer** [comment]
Looks like this will hopefully work now. Once it's done I'll merge and rebase it against it for my other pull.

**Pull request merged**